examples/WebRadioDemo/WebRadioDemo.ino

```diff
@@ -0,0 +1,113 @@
+/**
+  Wiring:
+  --------------------------------
+  | VS1053  | ESP8266 |  Other   |
+  --------------------------------
+  |   SCK   |   D5    |    -     |
+  |   MISO  |   D6    |    -     |
+  |   MOSI  |   D7    |    -     |
+  |   XRST  |    -    |    -     |
+  |   CS    |   D1    |    -     |
+  |   DCS   |   D0    |    -     |
+  |   DREQ  |   D3    |    -     |
+  |   5V    |    -    |   VCC    |
+  |   GND   |    -    |   GND    |
+  --------------------------------
+
+  Dependencies:
+  -VS1053 library by baldram (https://github.com/baldram/ESP_VS1053_Library)
+  -ESP8266Wifi
+
+  Instructions:
+  -Build the hardware
+  -Set the station in this file
+  -Upload the program
+
```

Comment:
One more thing. When look to [README.md](https://github.com/vincegellar/Simple-Radio-Node/commit/560a062bfb98ba1f1948066ddb3924e3e2b49e7f) of your project, it is said that IwIP2 is required (even [here](https://github.com/vincegellar/Simple-Radio-Node/blob/master/sr_node/sr_node.ino) it is still 1.4 so it looks like some inconsistency).
In case the correct one is IwIP Variant: v2.0 Higher Bandwidth, then please above platformio.ini also adjust and change build_flags to this:
```
build_flags = -D PIO_FRAMEWORK_ARDUINO_LWIP2_HIGHER_BANDWIDTH
```

In case it should be

Reply:
Wow thank you. I've tested both variants, the v2.0 appeared to handle stream hickups better. Even then, never would have noticed that I forgot to update that part.